### PR TITLE
chore: add unixodbc-dev and libpq-dev to list of dependencies

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: ci/Dockerfile [hunk 1]
```diff
@@ -51,6 +51,7 @@ RUN apt update -yq && apt install -yq software-properties-common \
     libxext6 libxrender1 libxkbcommon-dev libxkbcommon-x11-dev libxcomposite1 libxtst6 \
     libxrandr2 libxcursor1 libxi6 libxcb-randr0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 \
     libxcb-render-util0 libxcb-shape0 libxcb-render0 libxcb-xinerama0 \
+    unixodbc-dev \
```
Comment (jakubgs) [inline ci/Dockerfile:54]: Move them to where the rest of dev packages are:
image

**Review by jakubgs — CHANGES_REQUESTED** [review]
I will need to build this and push it. It doesn't build by itself because it's too slow and big.

**Comment by jakubgs** [comment]
Could you also fix this typo? It should be `s3cmd` and `file`:
https://github.com/status-im/status-desktop/blob/20c1544ec4e786bd3bf4cffbf9f3d65469978274/ci/Dockerfile#L43

**Comment by jakubgs** [comment]
I've pushed the image as `statusteam/nim-status-client-build:1.2.1-qt5.15.2`. Try it out.

**Comment by richard-ramos** [comment]
Done!

**Review by jakubgs — CHANGES_REQUESTED** [review]
Also, please don't change JUST `Dockerfile`, `scripts/ubuntu_build_setup.sh` also needs love.

**Comment by richard-ramos** [comment]
WTF. It's asking for libpq.so now

**Comment by richard-ramos** [comment]
Alright, fixed! @jakubgs if you have some time tomorrow, please publish statusteam/nim-status-client-build:1.2.2-qt5.15.2

**Comment by jakubgs** [comment]
New image pushed.

**Review by caybro — COMMENTED** [review]
Looks good!

**Pull request merged**